velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -134,27 +192,40 @@ std::tuple<std::string, bool> WindowFuzzer::generateFrameClause() {
         core::WindowNode::BoundType::kFollowing,
         core::WindowNode::BoundType::kUnboundedFollowing};
   } else {
-    startBoundOptions = {
-        core::WindowNode::BoundType::kUnboundedPreceding,
-        core::WindowNode::BoundType::kCurrentRow};
-    endBoundOptions = {
-        core::WindowNode::BoundType::kCurrentRow,
-        core::WindowNode::BoundType::kUnboundedFollowing};
+    // If it is a range frame and sorting keys size is 1, then that means
+    // k-range frames should be allowed for random selection
+    if (isSingleSortingKey) {
+      startBoundOptions = {
+          core::WindowNode::BoundType::kUnboundedPreceding,
+          core::WindowNode::BoundType::kPreceding,
+          core::WindowNode::BoundType::kCurrentRow,
+          core::WindowNode::BoundType::kFollowing};
+      endBoundOptions = {
+          core::WindowNode::BoundType::kPreceding,
+          core::WindowNode::BoundType::kCurrentRow,
+          core::WindowNode::BoundType::kFollowing,
+          core::WindowNode::BoundType::kUnboundedFollowing};
+    } else {
+      startBoundOptions = {
+          core::WindowNode::BoundType::kUnboundedPreceding,
+          core::WindowNode::BoundType::kCurrentRow};
+      endBoundOptions = {
+          core::WindowNode::BoundType::kCurrentRow,
+          core::WindowNode::BoundType::kUnboundedFollowing};
+    }
   }
 
   // End bound option should not be greater than start bound option as this
   // would result in an invalid frame.
   auto startBoundIndex = boost::random::uniform_int_distribution<uint32_t>(
       0, startBoundOptions.size() - 1)(rng_);
-  auto endBoundMinIdx = std::max(0, static_cast<int>(startBoundIndex) - 1);
+  auto endBoundMinIdx = std::max(0, static_cast<int>(startBoundIndex - 1));
   auto endBoundIndex = boost::random::uniform_int_distribution<uint32_t>(
       endBoundMinIdx, endBoundOptions.size() - 1)(rng_);
-  auto frameStart = frameBound(startBoundOptions[startBoundIndex]);
-  auto frameEnd = frameBound(endBoundOptions[endBoundIndex]);
+  auto frameStart = frameBound(startBoundOptions[startBoundIndex], "k0");
```

Comment:
nit: make "k0" and "k1" `static constexpr std::string` members in WindowFuzzer class and use them directly inside frameBound(). frameBound() doesn't need to receive an argument of columnName then, since it's not needed for other frame types.